rbac/server/api/main.py

```diff
@@ -16,6 +16,7 @@
 
 import asyncio
 from signal import signal, SIGINT
+from os import getenv
```

Comment:
Sorry knit pick here, but can you put above signal import for alphabetical, since we are trying to do that now?

Reply:
no prob, I'll move it